features/networking/ovn_ipsec.feature

```diff
@@ -157,3 +157,68 @@ Feature: OVNKubernetes IPsec related networking scenarios
     # Example ESP packet un-encrypted will look like 16:37:16.309297 IP ip-10-0-x-x.us-east-2.compute.internal > ip-10-0-x-x.us-east-2.compute.internal: ESP(spi=0xf50c771c,seq=0xfaad)
     And the output should match: 
        | <%= cb.workers[0].name %>.* > <%= cb.workers[1].name %>.*: ESP |
+       
+  # @author weliang@redhat.com
+  # @case_id OCP-40569
+  @admin
+  Scenario: Allow enablement/disablement ipsec at runtime
+    Given the env is using "OVNKubernetes" networkType
+    Given I store all worker nodes to the :workers clipboard
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    Given I have a project
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"] | <%= cb.workers[1].name %> |
+      | ["metadata"]["name"] | pod-worker1               |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=hello-pod |
+    And evaluation of `pod.ip_url` is stored in the :test_pod_worker1 clipboard
+    
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"]                 | <%= cb.workers[0].name %>                                                                     |
+      | ["metadata"]["name"]                 | pod-worker0                                                                                   |
+      | ["spec"]["containers"][0]["command"] | ["bash", "-c", "for f in {0..3600}; do curl <%= cb.test_pod_worker1 %>:8080 ; sleep 1; done"] |
+    Then the step should succeed
+    #Above command will curl "hello openshift" traffic every 1 second to worker1 test pod which is expected to cause ESP traffic generation across those nodes
+    And a pod becomes ready with labels:
+       | name=hello-pod |
+
+    #Enable ipsec through CNO
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with:
+      | {"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":{}}}}} |  
```

Comment:
any idea how long the conversion takes to complete?  should we add a small delay?

Reply:
The step of "Given I wait up to 60 seconds for the steps to pass:" will wait for enable/disable ipsec to take effect